lib/6to5/generation/whitespace.js

```diff
@@ -11,15 +11,17 @@ Whitespace.prototype.getNewlinesBefore = function (node) {
   var startToken;
   var endToken;
   var tokens = this.tokens;
+  var token;
+
+  for (var i = 0; i < tokens.length; i++) {
```

Comment:
@chicoxyzzy because v8 will do it for you: http://mrale.ph/blog/2014/12/24/array-length-caching.html